framework/decode/custom_vulkan_struct_to_json.cpp

```diff
@@ -250,6 +250,8 @@ void FieldToJson(nlohmann::ordered_json&                              jdata,
             case VK_PIPELINE_EXECUTABLE_STATISTIC_FORMAT_FLOAT64_KHR:
                 jdata["f64"] = decoded_value.f64;
                 break;
+            case VK_PIPELINE_EXECUTABLE_STATISTIC_FORMAT_MAX_ENUM_KHR:
+                GFXRECON_LOG_WARNING("Invalid format: VK_PIPELINE_EXECUTABLE_STATISTIC_FORMAT_MAX_ENUM_KHR");
```

Comment:
It looks a bit stupid to have these max_enum cases but a default handler would mask future enumerants that we wouldn't want to silently suppress.